localized_recurrence/models.py

```diff
@@ -3,6 +3,7 @@
 
 from dateutil.relativedelta import relativedelta
 from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
```

Comment:
This looks good. I typically import that decorator directly from [`six`](http://pythonhosted.org//six/#six.python_2_unicode_compatible)